server/monitor/src/main/java/org/apache/accumulo/monitor/rest/replication/ReplicationResource.java

```diff
@@ -149,6 +149,12 @@ public List<ReplicationInformation> getReplicationInformation()
       });
     }
 
+    //Ensure replication table is online before attempting to create BatchScanner
+    if (!ReplicationTable.isOnline(client)) {
+        log.error("Replication table is offline");
```

Comment:
I suspect the formatter will add a space when it builds. I enabled a build on this PR, so we'll find out.

The message can also be a debug instead of an error, since this is probably a normal situation (it's not a problem if the user clicks the page).

```suggestion
    // Ensure replication table is online before attempting to create BatchScanner
    if (!ReplicationTable.isOnline(client)) {
        log.debug("Replication page requested, but replication table is offline");
```

Reply:
```suggestion
    //Ensure replication table is online before attempting to create BatchScanner
    if (!ReplicationTable.isOnline(client)) {
        log.error("Replication table is offline");
```

Reply:
@ctubbsii In the Monitor, the Replication page already indicates "Replication table is offline". Is this sufficient?

Reply:
If that's already the case, then nothing needs to be done in that regard.

Reply:
Maybe better as a separate issue, but is that how the message reads? It may be more informative if the message indicated something along the lines of:
 
```Replication is disabled - the replication table is offline```

My thinking is that not using replication is more common. And having the replication off-line is the expected (desired) state.  Clarifying that may help if someone somehow interrupted off-line as bad.  Only people using replication should care.